.github/workflows/continuous-integration-unit-php.yml

```diff
@@ -24,12 +24,25 @@ jobs:
     continue-on-error: ${{ matrix.experimental == true }}
     strategy:
       matrix:
-        php: ['7.4', '7.0', '5.6']
+        php: ['7.3', '7.0', '5.6']
         wp: ['latest']
+        multisite: [false]
+        coverage: [false]
         include:
           - php: '8.0'
             wp: 'trunk'
```

Comment:
This makes me wonder: should we add a test to run against 7.4 & trunk? That would be `experimental:  false`

Reply:
Sounds good. Should this be included in this PR? Might be cool to run the e2e tests again trunk as well....

Reply:
7.4 & trunk can be in this PR, while wee're at it.

Agreed on E2E, but that is best left for a separate PR, as it probably requires some  more work.

Reply:
Fixed